nifi-nar-bundles/nifi-solr-bundle/nifi-solr-processors/src/main/java/org/apache/nifi/processors/solr/GetSolr.java

```diff
@@ -138,10 +168,11 @@ protected void init(final ProcessorInitializationContext context) {
         descriptors.add(SOLR_TYPE);
         descriptors.add(SOLR_LOCATION);
         descriptors.add(COLLECTION);
+        descriptors.add(RETURN_TYPE);
+        descriptors.add(RECORD_WRITER);
         descriptors.add(SOLR_QUERY);
-        descriptors.add(RETURN_FIELDS);
-        descriptors.add(SORT_CLAUSE);
```

Comment:
Is it safe to remove an existing property? The existing code should not sort result anyway, or should store last sorted field value to paginate properly when docs with the same date split more than one page. So I think it's safe..

Reply:
This should be save as the sorting only affects documents indexed after lastEndDate (documents indexed earlier are excluded by filter query)